src/app/beer_garden/api/stomp/__init__.py

```diff
@@ -22,11 +22,18 @@
 def run(ep_conn):
     global conn
```

Comment:
Any way we can avoid global variables like this? That way it is easier to reuse.